src/http/cities-http.js

```diff
@@ -0,0 +1,19 @@
+import * as citiesCore from '../core/cities-core';
+import {createJsonRoute} from '../util/express';
+import {assert} from '../validation';
+
+const getCities = createJsonRoute(function(req, res) {
+  const citiesParams = assert({
+    cityId: req.query.id,
+    cityName: req.query.name,
+  }, 'citiesParams');
+
+  return citiesCore.getCities(citiesParams)
+    .then(result => result.rows)
+    .catch(err => undefined);
```

Comment:
I think it's better to leak the exception here than hide it. Otherwise it can lead to hard to debug bugs. There should be error handler middleware that catches all unexpected error and handles them appropriately